lib/rm.js

```diff
@@ -25,7 +24,8 @@ function content (cache, integrity) {
 
 module.exports.all = all
 
-function all (cache) {
+async function all (cache) {
   memo.clearMemoized()
-  return rimraf(path.join(cache, '*(content-*|index-*)'))
+  const paths = await glob(path.join(cache, '*(content-*|index-*)'), { silent: true, nosort: true })
```

Comment:
Was rimraf using glob under the hood to parse that second arg?

Reply:
exactly that, yes. `rimraf` does a glob internally and deletes the results